tasks.py

```diff
@@ -63,3 +64,40 @@ def process_send_notifications():
         insert_q2 = construct_mail_sent_query(SYSTEM_EMAIL_GRAPH, bestuurseenheid_graph, bericht['bericht']['value'], email['uuid'])
         # helpers.log(insert_q2)
         update(insert_q2)
+
+        
+def send_confirmation_notifications():
+    """
+    Check for messages in berichtencentrum received from kalliope and send email to dossierbehandelaar letting them know the email has arrived succesfully
+
+    :returns ?:
+    """
+    helpers.log("fetching messages that need a confirmation notification to be sent ...")
+    q = find_kalliope_mail(PUBLIC_GRAPH, 70)
```

Comment:
Shouldn't the `70` be configurable as an environment variable ?

Reply:
Yess Replaced this by MAX_AGE env

Reply:
Might be useful to have a different value than the one used for the other notification no ?